app/views/directives/_warnings-popover.html

```diff
@@ -1,6 +1,6 @@
 <span ng-if="content">
   <span
-    dynamic-content="{{content}}"
+    dynamic-content="{{content | middleEllipses:350:'...<br>...'}}"
```

Comment:
I agree!  Unfortunately, using the html entity caused the second ellipsis to take up its own line in the popover, so I settled for the less semantic three periods.  :(